infinitest-eclipse/src/main/java/org/infinitest/eclipse/trim/ColorSettings.java

```diff
@@ -0,0 +1,25 @@
+package org.infinitest.eclipse.trim;
+
+import org.eclipse.swt.*;
+
+public class ColorSettings {
```

Comment:
We should probably also make other colors configurable:
* the Passing colors used in VisualStatusPresenter.setPassingColors()
* the WorkspaceErrors colors used in VisualStatusPresenter.statusChanged(WorkspaceStatus)